provision/roles/ci_nfs/server/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Install nfs-utils for NFS sharing
+  when: setup_nfs
+  become: true
+  yum: name=nfs-utils state=present
+  tags:
+  - ci_nfs
+  - test
+
+- name: Create NFS mount directory
+  when: setup_nfs
+  become: true
+  file:
+    path: /jenkins
+    state: directory
+    mode: 0777
+  tags:
+  - ci_nfs
+  - test
+
+- name: Add entry in /etc/exports to configure NFS mount
+  when: setup_nfs
+  become: true
+  lineinfile:
+    destfile: /etc/exports
+    line: "/jenkins *(rw,sync,no_subtree_check,all_squash,anonuid=0,anongid=0)"
```

Comment:
Can you use the `mount` module instead as used in pre-requisites.yml

I and @dharmit actually debugged on that and the mount module can add entries in /etc/exports.

Reply:
Umm, I'm not sure. What we debugged was `mount` module removing the directory if we ask it to remove mount point entry from `/etc/fstab`. I'm not sure 100% sure if the module lets you play with `/etc/exports`.

Reply:
keeping as it is, I could not find anything related to `/etc/exports` in [mount module documentation](https://docs.ansible.com/ansible/2.6/modules/mount_module.html)